### DISCUSSION TITLE
Filter not paid received documents

### DISCUSSION THREAD

**Discussion by aster94** [fattureincloud/api#171] [Q&A] [answered]
Hello,

I am in the process of migrating from V1 to V2.
In the old version for received documents (acquisti) there was a field in the request (saldato) to filter only not paid documents. I can't find anything similar in V2. Can you point me where is it or how to work around it?

Thanks

**Comment by emajo** [comment, marked as answer]
Hi @aster94 to obtain only the not paid documents you have to filter for `next_due_date != null`.

Thanks,
Emanuele.

**Reply by aster94** [reply]
Thanks!
